src/TritonCTS/src/third_party/CKMeans/clustering.cpp

```diff
@@ -481,11 +353,10 @@ void clustering::minCostFlow (const vector<pair<float, float>>& means, unsigned
 
     for (unsigned i = 0; i < f_nodes.size(); ++i)
         node_map[f_nodes[i]] = make_pair(i, make_pair(-1,-1));
-//    for (unsigned i = 0; i < s_nodes.size(); ++i)
-//        for (unsigned j = 0; j < CAP; ++j)
-//            node_map[s_nodes[i][j]] = make_pair(-1, make_pair(i,j));
+
     for (unsigned i = 0; i < c_nodes.size(); ++i)
         node_map[c_nodes[i]] = make_pair(-1, make_pair(i,0));
+
     node_map[s] = make_pair(-2, make_pair(-2, -2));
```

Comment:
make_pair(make_pair is begging for a structure. make_pair is a lazy substitute and has no names that convey the type or use of the values